MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -492,10 +505,11 @@ object VisionManager : ARDataProvider {
     }
 
     private fun requestCoreUpdate() {
-        val lastCoreUpdateStartTime = System.currentTimeMillis()
+        coreUpdateThreadHandler.postDelayed(
+                { requestCoreUpdate() },
+                CORE_UPDATE_DELAY_MILLIS
```

Comment:
As far as I remember we should update the core every 30 milliseconds minus time for method execution?

Reply:
It posts the next update for 30 FPS and then requests update, so the FPS is the actual 30 FPS